### PR TITLE
Add Terraform resources

### PR DESCRIPTION
**Pull request by aknysh**
## What

* Terraform module for AWS RDS

## Why

* Generic module to provision RDS instances (MySQL, Postgres, Oracle, SQL Server) on AWS

### CODE REVIEW

**Comment by osterman** [comment]
- [ ] Add `README.md`
- [ ] Fix `LICENSE` https://github.com/cloudposse/tf_rds/blob/master/LICENSE#L189

**Comment by aknysh** [comment]
Added `README.md`
 Fixed `LICENSE`

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  password                    = "${var.database_password}"
+  port                        = "${var.database_port}"
+  engine                      = "${var.rds_engine_type}"
+  engine_version              = "${var.rds_engine_version}"
+  instance_class              = "${var.rds_instance_class}"
+  allocated_storage           = "${var.rds_allocated_storage}"
+  vpc_security_group_ids      = ["${aws_security_group.default.id}"]
+  db_subnet_group_name        = "${aws_db_subnet_group.default.name}"
+  parameter_group_name        = "${aws_db_parameter_group.default.name}"
+  multi_az                    = "${var.rds_is_multi_az}"
```
Comment (osterman) [inline main.tf]: Why introduce a new `rds_is_` prefix? It's not used consistently. Prefer to mimic the original variable names where we simply pass them through to the underlying resource. If we do a data transformation on the variable, then we might want to call it something different.

**Comment by aknysh** [comment]
Fixed variable names

**Pull request merged**